ASCIIEngine/Core.cs

```diff
@@ -26,42 +25,21 @@ public void DoStep()
 
             foreach (var o in objects)
             {
-                o.HasChanged = false;
                 o.Step();
             }
 
-            CheckForCollisions(objects
-                .Where(o => o.HasChanged));
+            CollisionHandler.ResolveCollisions(objects.Where(o => o.HasChanged || o.IsStatic));
```

Comment:
I think we should remove HasChanged property, because if two object collide and don't move, then their collision will be ignored in further steps, and their OnCollision methods won't be triggered.

Reply:
removed